src/java.base/share/classes/jdk/crac/CheckpointException.java

```diff
@@ -36,16 +36,8 @@ public class CheckpointException extends Exception {
      * Creates a {@code CheckpointException}.
```

Comment:
I probably misclicked "Add comment",

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Throwable.html#%3Cinit%3E(java.lang.String,java.lang.Throwable,boolean,boolean):
> Subclasses of Throwable should document any conditions under which suppression is disabled and document conditions under which the stack trace is not writable.

The javadoc should specify the stack trace is not writeable, here and for other exceptions